zookeeper-server/src/main/java/org/apache/zookeeper/ZooKeeperMain.java

```diff
@@ -282,45 +283,19 @@ public ZooKeeperMain(ZooKeeper zk) {
     void run() throws IOException, InterruptedException {
         if (cl.getCommand() == null) {
             System.out.println("Welcome to ZooKeeper!");
+            System.out.println("JLine support is enabled");
 
-            boolean jlinemissing = false;
-            // only use jline if it's in the classpath
-            try {
-                Class<?> consoleC = Class.forName("jline.console.ConsoleReader");
```

Comment:
This part has been dropped.
So we can't work anymore without jline

Reply:
yes, The code looks cluttered if we use reflection in jline3 (The apis are very different from that of jline 2.). BTW, I think jline3 should be default use in zkCli for its new features, just like histroy, tab key, prompt, etc. These new features are very user-friendly to the zkCli tool. Do you have any other ideas？
Thank you very much! @eolivelli

Reply:
Ya, my idea was to remove reflection as well since having jline as default seems like the way forward, given that it makes the zkCli very user friendly.

Reply:
Thank you @surajj-naik. I will follow you to learn more valuable ideas about more projects continuously.